src/Components/test/E2ETest/Infrastructure/WebDriverExtensions/BasicTestAppWebDriverExtensions.cs

```diff
@@ -8,7 +8,10 @@ internal static class BasicTestAppWebDriverExtensions
     {
         public static IWebElement MountTestComponent<TComponent>(this IWebDriver browser) where TComponent : IComponent
         {
-            var componentTypeName = typeof(TComponent).FullName;
+            var componentType = typeof(TComponent);
+            var componentTypeName = componentType.Assembly == typeof(BasicTestApp.Program).Assembly ?
+                componentType.FullName :
+                componentType.AssemblyQualifiedName;
```

Comment:
Interesting. Why is this change needed?

Reply:
Ah I see it's related to `ProtectedBrowserStorageUsageComponent` being in the server project, however I'm still unsure of the necessity as per question https://github.com/dotnet/aspnetcore/pull/25557/files#r482820977.